xblock/core.py

```diff
@@ -23,6 +23,7 @@
 from xblock.fields import Field, List, Reference, ReferenceList, Scope, String
 from xblock.internal import class_lazy
 from xblock.plugin import Plugin
+from xblock.utils.helpers import is_pointer_tag, load_definition_xml
```

Comment:
Unlike in other repos, the `utils`, package in xblock is not meant to hold core helper methods. It actually holds _edx-platform developer utilities_ (e.g. StudioEditableMixin)-- the name is historical. `xblock` doesn't ever import from `xblock.utils`, and it would help the code stay simple if we kept it that way.

Could you move the new helpers to a new file `xblock/xml.py` ?